web/src/ApiButton.test.tsx

```diff
@@ -37,18 +37,23 @@ const buttonInputsAccessor = accessorsForTesting(
   localStorage
 )
 
-function wrappedMount(e: JSX.Element) {
-  return mount(
+function Wrapper(props: PropsWithChildren<{ setError?: () => {} }>) {
+  return (
     <MemoryRouter>
-      <tiltfileKeyContext.Provider value="test">
-        <SnackbarProvider>{e}</SnackbarProvider>
-      </tiltfileKeyContext.Provider>
+      <HudErrorContextProvider setError={props.setError ?? (() => {})}>
+        <tiltfileKeyContext.Provider value="test">
+          <SnackbarProvider>{props.children}</SnackbarProvider>
+        </tiltfileKeyContext.Provider>
+      </HudErrorContextProvider>
     </MemoryRouter>
   )
 }
 
-function mountButton(b: UIButton) {
-  return wrappedMount(<ApiButton uiButton={b} />)
+function mountButton(b: UIButton, wrapperProps?: {}) {
+  return mount(<ApiButton uiButton={b} />, {
+    wrappingComponent: Wrapper,
+    wrappingComponentProps: wrapperProps,
```

Comment:
fun fact: today i learned about the `wrappingComponent` option in enzyme and it's so convenient! it means we can mount the component we care about and change its props (which can only be done on the component that `mount()` returns) _and_ wrap it in the providers we need it to be wrapped in. 🧠